apps/predict-ui/src/app/agent.tsx

```diff
@@ -25,23 +25,51 @@ const AgentContent = styled.div`
 const AgentLoader = () => (
   <Flex vertical gap={24}>
     <AgentContent>
-      <LoaderCard />
+      <Card>
+        <Row gutter={24} align="middle" justify="space-between" className="m-0">
+          <Col lg={12} sm={12} xs={24} className="p-0">
+            <Skeleton.Input active size="small" />
+          </Col>
+          <Col lg={12} sm={12} xs={24} className="p-0">
+            <Skeleton.Input active size="small" />
+          </Col>
+        </Row>
+      </Card>
+
+      <Card>
+        <Skeleton.Input active />
+        <Row gutter={[24, 24]} align="middle" justify="space-between" className="m-0">
+          {[0, 1, 2, 3, 4, 5].map((index) => (
+            <Col key={index} lg={8} sm={12} xs={24} className="p-0">
+              <Skeleton.Input active style={{ width: 196 }} />
+            </Col>
+          ))}
+        </Row>
+      </Card>
     </AgentContent>
   </Flex>
 );
 
 const AgentError = () => (
   <Flex vertical gap={24}>
     <AgentContent>
-      <LoadingError />
+      <ErrorState
+        title="Error loading data"
+        description="Something went wrong while loading data. Please, try closing the app."
```

Comment:
we shouldn't mention app here, as we can't be sure only Pearl is the consumer of this UI, no?